src/main/java/com/minecolonies/api/items/ModItems.java

```diff
@@ -122,4 +162,83 @@ private ModItems()
          * Intentionally left empty.
          */
     }
+
+    @NotNull
+    public static Item[] getAllFoods()
+    {
+        return new Item[] {
+          muffin_dough,
```

Comment:
Whilst I understand why this exists, for the creative tab. Wouldn't it be better to name the method something else or split the method into 2 separate methods which return ingredients and actual food separately?

In my food tags PR I also created a method for fetching all food but I explicitly did not include the dough items etc because they're not actual "food".

I feel like 2 separated lists with the ingredients and actual food are better.